Libplanet/Action/IAction.cs

```diff
@@ -336,5 +351,22 @@ public interface IAction
         /// <see cref="Render(IActionContext, IAccountStateDelta)"/> method
         /// with redrawing the graphics on the display at the finish.</remarks>
         void Unrender(IActionContext context, IAccountStateDelta nextStates);
+
+        /// <summary>
+        /// Does things that should be undone right after this action is
+        /// invalidated (mostly due to a block which this action has belonged
+        /// to becoming considered a stale).
```

Comment:
How about explaining this method like the following way:

```suggestion
        /// Does the similar things to <see cref="Unrender(IActionContext, IAccountStateDelta)"/>,
        /// except that this method is invoked when see cref="Execute(IActionContext)"/> method
        /// has terminated with an exception.
```